html/user/edit_passwd_action.php

```diff
@@ -45,7 +46,8 @@
 }
 
 $passwd_hash = md5($passwd.$user->email_addr);
-$result = $user->update("passwd_hash='$passwd_hash'");
+$database_passwd_hash = password_hash( $passwd_hash, PASSWORD_DEFAULT);
+$result = $user->update("passwd_hash='$database_passwd_hash'");
```

Comment:
[Here](https://github.com/BOINC/boinc/pull/2401/files#diff-2ebf2ee712812c0682531ca5ebf7c713R73) you escape the password hash before its use as part of an SQL clause. If you think it's required, it should be done here as well.

Reply:
After reviewing, the hash does not need to be escaped so I have removed it from the referenced line.